news/1082-unset-variables

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* The shell-based installers now sanitize additional environment variables for consistency with security best practices. (#1082)
```

Comment:
```suggestion
* Unset additional environment variables in shell-based installers to avoid accidental loading of external libraries. (#1082)
```

Since I don't think this is really exploitable, I would avoid raising the specter or a security vulnerability here. However, a user could accidentally shoot themselves in the foot here.